pages/common/mysqldump.md

```diff
@@ -10,3 +10,11 @@
 - Restore a backup, user will be prompted for a password:
 
 `mysql -u {{user}} --password -e "source {{filename.sql}}" {{database_name}}`
+
+- Backup all databases with compression, user will be prompted for a password:
```

Comment:
I think we can leave out the compression part. It is an external command and is not relevant to this page. And after that, please mention the "redirect to a file" part.

Reply:
Do you mean to exclude the piping to gzip part in the command too? I think it's a common use case that you want to compress your backup files and because `mysqldump` doesn't provide that option by default that an example of it is warranted.

Reply:
Yes, but that is a separate command. We usually leave that up to the users and just keep the pages focused on a single command.

Reply:
OK, I've removed the compression part and updated the PR